ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/mapping/properties/AvgivendeSystemHandler.java

```diff
@@ -0,0 +1,31 @@
+package no.difi.meldingsutveksling.ks.mapping.properties;
+
+import com.google.common.base.Objects;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.ks.Forsendelse;
+
+public class AvgivendeSystemHandler extends PropertyHandler<Forsendelse.Builder> {
+
+    public AvgivendeSystemHandler(IntegrasjonspunktProperties properties) {
+        super(properties);
+    }
+
+    @Override
+    public Forsendelse.Builder map(Forsendelse.Builder builder) {
+        builder.withAvgivendeSystem(super.getProperties().getNoarkSystem().getType());
+        return builder;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00122)